inst/@sym/horzcat.m

```diff
@@ -62,7 +62,7 @@
           'return sp.Matrix.hstack(*_proc),'
           };
 
-  varargin = sym(varargin);
+  varargin = cell_array_to_sym(varargin);
```

Comment:
mm, seems i misinterpret a little why you change sym to cell_array_to_sym, yes it don't works, but we can call normally the classdef with cells but the private function for some reason can't be called:
```
octave:1> sym({'f'})
error: 'cell_array_to_sym' undefined near line 210 column 13
error: called from
    sym at line 210 column 11
```
i try adding the function as private in the classdef but seems don't is supported for octave...